src/components/Transaction.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { useEffect } from 'react';
+import { useState } from 'react'
+import { getAllCheckingAccounts, transaction } from '../api';
+
+export const Transaction = (props) => {
+
+
+    const [accounts,setAccounts] = useState([]); 
+    const [loading,setLoading] = useState(true); 
+    const [transactionValues,setTransactionValues] = useState({
+        amount:'',
+        currentAccount:''
+    }); 
+    const [error,setError] = useState(false); 
+    const [success,setSuccess] = useState(false); 
```

Comment:
_errorMessage_ and _successMessage_ are more suitable names for these states. They provide more information about the context where you might use them.

Reply:
You are right!